demo/package-lock.json

```diff
@@ -8172,6 +19514,15 @@
       "integrity": "sha1-1cdSgl5TZ+eG944Y5EXqIjoVWVI=",
       "dev": true
     },
+    "string_decoder": {
```

Comment:
this change seems to reorder a lot of stuff in here, would it be possible to reduce the amount of change in the lockfile? :sweat_smile:

Reply:
It's done.

Reply:
still sais 11500 changes to the lockfile for me

Reply:
It changes the version of every package, including those of our root deps that can't be the intention 😅 you are adding things to _demo_ and things are changing in the root folder

Reply:
Got it, please check again. @JoviDeCroock

Reply:
> It changes the version of every package, including those of our root deps that can't be the intention 😅 you are adding things to _demo_ and things are changing in the root folder

Done.